active-rfcs/0004-api-versioning.md

```diff
@@ -32,34 +31,33 @@ We will manage our API through semantic versioning, being the client who defines
 curl -i -H " Accept: application/vnd.wawmuzik.v1+json" -H "Content-Type: application/json" http://app.wawmuzik.com/auth/login
 ```
 
-the response will be come with this type of header:
+the response will come with this type of header:
 
 ```bash
 Content-Type: application/vnd.wawmuzik.v1+json
 ```
 
-In backend, in our routers, we will need to check this header before do something in the ednpoint, so we will need to make a Factory that manage this behaviour.
-
-For now, requests that don't have this header will be redirected to the first version of our API. This approach will be for now, because in the future, when we will remove this 1 version, requests that don't have this header, will be redirected to the latest version
+In backend, in our routers, we will need to check this header before doing anything in the endpoint, so we will have to make a Factory that manages this behaviour.
```

Comment:
```suggestion
In the backend, in our routers, we will need to check this header before doing anything in the endpoint. This will be managed by a Factory, which must be implemented soon.
```